GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -100,6 +105,14 @@ public static void Report(Exception exception, bool isTerminating)
                 LogError(exception, isTerminating);
             }
 
+            if (exception is ExternalOperationException externalOperationException
+                && externalOperationException.InnerException is { }
+                && externalOperationException.InnerException.Message.Contains(_dubiousOwnershipSecurityConfigString))
```

Comment:
Git commands that handles Git errors will not raise this exception.
I guess that there will be at least one command in the revision refresh scenario that will though.

I wish there were a safer way to detect the issue, but this I assume this is good enough.